src/configs/browserify-mocha-sinon-chai.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+    env: { // https://eslint.org/docs/user-guide/configuring#specifying-environments
+        browser: true,
+        commonjs: true,
+        es6: true,
+        mocha: true,
+    },
+    globals: {
+        it: true,
+        sinon: true,
+    },
+    parserOptions: { // even though we plan to use babel-eslint as the parser, eslint currently requires this parserOptions set for some reason
+        sourceType: 'module',
+        ecmaFeatures: {
+            jsx: true,
+            impliedStrict: true,
+            classes: true,
+        },
+    },
+    extends: [
+        './src/parser/babel-eslint.js',
+        './src/rules/index.js',
+        './src/plugin/compat--browserify.js',
+        './src/plugin/filenames.js',
+        './src/plugin/import--browserify.js',
+        './src/plugin/jsdoc.js',
+        './src/plugin/jsx-a11y.js',
+        './src/plugin/react.js',
+    ],
+    rules: {
+        'max-nested-callbacks':     'off',      // our tests are deep like our love
+        'newline-per-chained-call': 'off',      // keep our tests a bit more compact
+        'no-invalid-this':          'off',      // the context of `this` inside mocha functions is fine. Go away.
+        'no-unused-expressions':    'off',      // allows expect(true).to.be.true;
+        'no-restricted-properties': ['error', { // disallow certain properties on certain objects
+            object: 'describe',
+            property: 'only',
+        }, {
+            object: 'context',
+            property: 'only',
+        }, {
+            object: 'it',
+            property: 'only',
+        }, {
+            object: 'describe',
+            property: 'skip',
```

Comment:
We're throwing errors for skipped tests now? Not just warnings?

Reply:
https://github.com/FoxSportsAustralia/fs-default-project-config/blob/master/resources/.eslintrc-standards--test-override.js#L19-L20 we always did 😄